Pod/Classes/ValueStepper.swift

```diff
@@ -124,6 +118,7 @@ private enum Button: Int {
     
     public required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Setting a value as the `intrinsicContentSize` in the subclass allows IB to automatically infer the width and height constraints without actually specifying them.
<img width="881" alt="screen shot 2016-02-20 at 5 06 45 pm" src="https://cloud.githubusercontent.com/assets/3658887/13197652/7fbcf178-d7f4-11e5-87d9-551d474fc7a9.png">